audb/core/api.py

```diff
@@ -51,16 +51,21 @@ def available(
                 # see https://github.com/audeering/audbackend/issues/132
                 for p in backend._repo.path:
                     name = p.name
-                    for version in [str(x).split("/")[-1] for x in p / "db"]:
-                        databases.append(
-                            [
-                                name,
-                                repository.backend,
-                                repository.host,
-                                repository.name,
-                                version,
-                            ]
-                        )
+                    try:
+                        for version in [str(x).split("/")[-1] for x in p / "db"]:
+                            databases.append(
+                                [
+                                    name,
+                                    repository.backend,
+                                    repository.host,
+                                    repository.name,
+                                    version,
+                                ]
+                            )
+                    except FileNotFoundError:
+                        # If the `db` folder does not exist,
+                        # we do not include the dataset
+                        pass
```

Comment:
Should one warn about this happening? I mean there is no logging. Not that it would help :-)

Afaics there are no other possible exception classes , as name and verions as well as the repo attributes will always have to be there. Or are there?

Reply:
I would not add a warning here. If we really want to inspect a repository for broken datasets, I would propose to add a special function for this.

There might be other exceptions, but maybe some of them are exceptions that a user should see. For this reason I try to avoid using `except Exception` and try to specify only the ones, that we are sure we want to catch.